README.md

```diff
@@ -3,6 +3,8 @@ rscpages: Curate your content on RStudio Connect
 
 <!-- badges: start -->
 
+[![CRAN status](https://www.r-pkg.org/badges/version/rscpages)](https://CRAN.R-project.org/package=rscpages)
```

Comment:
You want a CRAN badge before we submit?

Reply:
Yep, at least one person during testing missed that it wasn't on cran, so the badge will show "not published" until it lands, at which point it should update.